src/components/flat-table/flat-table-cell/flat-table-cell.style.js

```diff
@@ -16,7 +16,7 @@ const StyledFlatTableCell = styled.td`
 
     > div {
```

Comment:
I don't think these will override the defaults applied for padding left and right (see screenshot). The unit tests pass as the styling is applied here https://github.com/Sage/carbon/blob/c547f4efdd035bb2dd0013b94c21bb38df8237c7/src/components/flat-table/flat-table.style.js#L56  I think you will need to raise the specificity like it is currently in `master` like so https://github.com/Sage/carbon/blob/6dba4e9ae48618e7d6ea9fbd6e9ffe6323aea730/src/components/flat-table/flat-table-cell/flat-table-cell.style.js#L29

![image](https://user-images.githubusercontent.com/44157880/114707654-61314580-9d22-11eb-8224-0ddbeb13edef.png)

Reply:
Good point, changed. Thanks!